src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/PersonsTest.kt

```diff
@@ -142,6 +143,65 @@ class PersonsTest: AbstractTest() {
         }
     }
 
+    /**
+     * Tests overriding vacation days for users via v/1/persons/{personId} -endpoint PUT method. Only admin-roles should be allowed to perform this action.
+     */
+    @Test
+    fun overrideVacationDaysTest() {
+        createTestBuilder().use { testBuilder ->
+            val person = testBuilder.userA.persons.getPersons(active = false).find { it.id == 2 }!!
+            val newPerson = Person(
+                id = person.id,
+                firstName = person.firstName,
+                lastName = person.lastName,
+                email = person.email,
+                monday = person.monday,
+                tuesday = person.tuesday,
+                wednesday = person.wednesday,
+                thursday = person.thursday,
+                friday = person.friday,
+                saturday = person.saturday,
+                sunday = person.sunday,
+                active = person.active,
+                unspentVacations = 32,
+                spentVacations = 37,
+                minimumBillableRate = person.minimumBillableRate,
+                language = person.language,
+                startDate = person.startDate
+            )
+            val updatedPerson = testBuilder.manager.persons.updatePerson(
+                personId = newPerson.id,
+                person = newPerson
+            )
+
```

Comment:
It is a good practice to re-find the REST object after updating to verify that the updated values are actually persisted.